jdkhttp/README.md

```diff
@@ -151,4 +151,155 @@ E.g. image:
 ### REST API
 ```sh
  http://<address>/rest/env/all
-```
\ No newline at end of file
+```
+
+### Update KJar at startup
+
+#### The prerequisites
+To have an updatable Kjar at startup and later, is mandatory to add two env vars 
+at the startup time and the presence of the specific jar in a Nexus repo. 
+
+Install a Nexus sonatype , and provide a maven-group to
+group together your maven repos needed by your project,
+copy the Nexus url, something like:
```

Comment:
The command to install nexus on openshift is one line, but how to configure volumes and settings to read from a external repo is mandatory if we describe how to update the kjar , otherwise who know how to setup correctly our code ?

Reply:
@danielezonca In general it is true. But I don't like these readmes at all to be honest. We have community docs for this. It is very hard to maintain and understand these readmes. I would suggest to move content of these readmes into community docs.